otelcollector/deploy/addon-chart/Readme.md

```diff
@@ -0,0 +1,22 @@
+### **Step 1: Go to addon-chart directory**
```

Comment:
Maybe we could also doc as a pre-req to follow the ARM template deployment instructions (but set azuremonitormetrics to false) to deploy the other artifacts first?

Reply:
Yes. I have a couple more instructions to add. Let me add  it and you can reapprove. Thanks!

Reply:
Updated the readme. pls take a look.